src/components/UserProfile/UserSettings/index.tsx

```diff
@@ -73,6 +75,14 @@ const UserSettings: React.FC = ({ children }) => {
     regex: RegExp;
     isMobile?: boolean;
   }> = ({ children, route, regex, isMobile = false }) => {
+    if (
+      route === '/settings/password' &&
+      !settings.currentSettings.localLogin &&
+      !hasPermission(Permission.MANAGE_SETTINGS)
+    ) {
+      return null;
+    }
```

Comment:
Let's try to keep the PR's scope focused on one thing if we can. Just keep it in mind for next time.